### PR TITLE
Add food category filter (using mock data)

### PR DESCRIPTION
**Pull request by DLi7077**
Implemented food category filter using mock data
We'll replace this mock data with food categories retrieved from the server

https://user-images.githubusercontent.com/51217000/236128820-d2bddcf4-3db5-4f0c-b22e-6956baa3597c.mp4

### CODE REVIEW

File: client/src/components/Filters/FoodCategoryFilter.js
Comment (DLi7077) [inline client/src/components/Filters/FoodCategoryFilter.js:1]: Casually dunks on Omer

File: client/src/components/Filters/PriceFilter.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ export default function PriceFilterToggler({ priceFilter, setPriceFilter }) {
   return (
     <ToggleButtonGroup
       value={selectedPrices}
-      style={{ backgroundColor: "hsl(35, 10%, 98%)" }}
+      style={{ backgroundColor: "hsl(35, 10%, 98%)", height: "fit-content" }}
```
Comment (DLi7077) [inline client/src/components/Filters/PriceFilter.js:12]: prevent toggle buttons from having changing height (mui is weird)

File: client/src/pages/Search/SearchHeader/RestaurantFilter.js
Comment (DLi7077) [inline client/src/pages/Search/SearchHeader/RestaurantFilter.js:1]: Houses all the filters

**Review by Remos96 — APPROVED** [review]
Looks good broski

**Pull request merged**